src/emitter.ts

```diff
@@ -323,6 +321,8 @@ export function emitWebIDl(webidl: Browser.WebIdl, flavor: Flavor) {
                     allEnumsMap[objDomType]) return objDomType;
                 // Name of a type alias. Just return itself
                 if (allTypeDefsMap.has(objDomType)) return objDomType;
+                // Known interface name but not for the target flavor
+                if (unfilteredInterfaceNames.has(objDomType)) return "never";
```

Comment:
not sure i see why this is needed over `object` (Which is the fallback for this method anyways).

Reply:
type X = Body | object;
var x: X = {}; // no error